### PR TITLE
Update Aurora MySQL dumpling command

### PR DESCRIPTION
**Pull request by fengou1**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: migrate-from-aurora-using-lightning.md [hunk 1]
```diff
@@ -79,12 +79,12 @@ summary: 了解如何使用 TiDB Lightning 从 Amazon Aurora MySQL 迁移全量
 1. 使用 Dumpling 导出表结构文件：
 
     ```
-    ./dumpling --host 127.0.0.1 --port 4000 --user root --password password --no-data --output ./schema --filter "mydb.*"
+    ./dumpling --host database-1.cedtft9htlae.us-west-2.rds.amazonaws.com --port 3306 --user root --password password --consistency none --no-data --output ./schema --filter "mydb.*"
```
Comment (lichunzhu) [inline migrate-from-aurora-using-lightning.md:82]: I think locking these tables for a short while is better.
```suggestion
    ./dumpling --host database-1.cedtft9htlae.us-west-2.rds.amazonaws.com --port 3306 --user root --password password --consistency lock --no-data --output ./schema --filter "mydb.*"
```

**Comment by fengou1** [comment]
1. Aurora MySQL does not support SUPR privileges allow to local table (consistent) so that we need --consistency none
2. Aurora MySQL use port 3306

**Comment by lichunzhu** [comment]
> 1. Aurora MySQL does not support SUPR privileges allow to local table (consistent) so that we need --consistency none
> 2. Aurora MySQL use port 3306

But we can lock tables in Aurora MySQL.

**Comment by fengou1** [comment]
> > 1. Aurora MySQL does not support SUPR privileges allow to local table (consistent) so that we need --consistency none
> > 2. Aurora MySQL use port 3306
> 
> But we can lock tables in Aurora MySQL.

Just check the privileges of Aurora MySQL, and you are right, we are able to lock the table.

File: migrate-from-aurora-using-lightning.md (hunk 1 above)
Comment (fengou1) [reply migrate-from-aurora-using-lightning.md:82]: Tried the command with --consistency lock, it works. To keep doc identify with TiDB Cloud https://docs.pingcap.com/tidbcloud/public-preview/migrate-from-aurora-bulk-import, shall we keep none?

File: migrate-from-aurora-using-lightning.md [hunk 2]
```diff
@@ -79,12 +79,12 @@ summary: 了解如何使用 TiDB Lightning 从 Amazon Aurora MySQL 迁移全量
...
     ```
 
     > **注意：**
     > 
-    > - 请根据实际情况设置数据源地址的相关参数和输出文件的路径。
+    > - 请根据实际情况设置数据源地址的相关参数和输出文件的路径。例如 `database-1.cedtft9htlae.us-west-2.rds.amazonaws.com` 是 Aurora MySQL 的地址
```
Comment (TomShawn) [inline migrate-from-aurora-using-lightning.md:87]:
```suggestion
    > - 请根据实际情况设置数据源地址的相关参数和输出文件的路径。例如 `database-1.cedtft9htlae.us-west-2.rds.amazonaws.com` 是 Aurora MySQL 的地址。
```

**Review by TomShawn — APPROVED** [review]

**Pull request merged**